sql/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java

```diff
@@ -653,11 +653,11 @@ public void testInsertFromSubQueryPartitionedTableCustomSchema() throws Exceptio
         ensureGreen();
         refresh();
 
-        execute("select * from custom.source order by zipcode");
+        execute("select * from custom.source order by name");
```

Comment:
ordering by zipcode is not deterministic as there are two rows with the same zipcode